server/athenian/api/controllers/miners/github/pull_request.py

```diff
@@ -208,11 +208,13 @@ async def _mine(cls,
         assert (updated_min is None) == (updated_max is None)
         time_from, time_to = (pd.Timestamp(t, tzinfo=timezone.utc) for t in (date_from, date_to))
         if pr_blacklist is not None:
-            pr_blacklist, ambiguous = pr_blacklist
-            if len(pr_blacklist) > 0:
-                pr_blacklist = PullRequest.node_id.notin_any_values(pr_blacklist)
+            pr_blacklist_map, ambiguous = pr_blacklist
+            if len(pr_blacklist_map) > 0:
+                pr_blacklist_keys = list(pr_blacklist_map)
```

Comment:
Again, we cannot change this in prod. These lines are literally written with my blood and sweat.